### PR TITLE
Function (Fibonacci number)

### PR DESCRIPTION
**Pull request by AdilkedirGITHUB**
Finished the exercise, please look at it. Thanks

### CODE REVIEW

File: students/Adil/Class_assignments/FizzBazz.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+for fizzbuzz in range(100):
+    if fizzbuzz % 3 == 0 and fizzbuzz % 5 == 0:
+        print('Fizz')
+        continue
+    elif fizzbuzz % 3 == 0:
+        print('Buzz')
+        continue
```
Comment (summerela) [inline students/Adil/Class_assignments/FizzBazz.py]: Why are you using continue statements here?

File: students/Adil/Class_assignments/Function_Fibonacci.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        print("Incorrect input")
+    # First Fibonacci number is 0
+    elif n==1:
+        return 0
+    # Second Fibonacci number is 1
+    elif n==2:
+        return 1
+    else:
+        return Fibonacci(n-1)+Fibonacci(n-2)
+for n in range (1,15):
```
Comment (summerela) [inline students/Adil/Class_assignments/Function_Fibonacci.py]: Does range(1,15) include 15?

File: students/Adil/Class_assignments/grid.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+print(rowCell)
+print(rowCell)
+print(rowCell)
+print(rowCell)
+print(colCell)
+print(rowCell)
+print(rowCell)
+print(rowCell)
+print(rowCell)
+print(colCell)
```
Comment (summerela) [inline students/Adil/Class_assignments/grid.py]: Hi Adil-

Check out [this link for some pointers](https://stackoverflow.com/questions/22584652/python-how-to-print-out-a-2d-list-that-is-formatted-into-a-grid) on how to solve this.

**Comment by AdilkedirGITHUB** [comment]
Hi Summer,
Thanks for the comments, no the range was supposed to be (0,15) but I was
getting some errors
and I end up changing the code in the following way.
# Function (Fibonacci number)

def Fibonacci(n):
    if n<0:
        print("Incorrect input")
    # First Fibonacci number is 0
    elif n==0:
        return 0
    # Second Fibonacci number is 1
    elif n==1:
        return 1
    else:
        return Fibonacci(n-1)+Fibonacci(n-2)
for n in range (0,15):
    print(Fibonacci(n))


Thanks

**Pull request merged**